src/app.ts

```diff
@@ -5,19 +5,34 @@ import session from "express-session";
 
 import { getAllDatabases } from "./controllers/TestController";
 
+const path = require('path');
+
 const app = express();
 
-nunjucks.configure('views', {
-    autoescape: true,
-    express: app
+nunjucks.configure([
+  "node_modules/govuk-frontend/dist",
+  "views"
```

Comment:
why double quoted?

Reply:
Good question

Reply:
because everything else in the file is single quoted